contrib/copy-tex/copy-tex.js

```diff
@@ -12,10 +12,8 @@ document.addEventListener('copy', function(event) {
         return;  // default action OK if no .katex-mathml elements
     }
     // Preserve usual HTML copy/paste behavior.
-    const html = [];
-    for (let i = 0; i < fragment.childNodes.length; i++) {
-        html.push(fragment.childNodes[i].outerHTML);
-    }
+    const html = fragment.childNodes.map(node => node.outerHTML);
```

Comment:
`fragment.childNodes` is a `NodeList` and it doesn't have `map` method. You can use `Array.prototype.map.call`.